src/main/scala/nl.knaw.dans.easy.validatebag/rules/metadata/package.scala

```diff
@@ -248,7 +281,8 @@ package object metadata extends DebugEnhancedLogging {
       files =>
         if (files.namespace == filesXmlNamespace) {
           debug("Rule filesXmlHasOnlyFiles has been checked by files.xsd")
-        } else {
+        }
```

Comment:
Hm, I guess I never ran the formatter on this code :-/

Reply:
Didn't even notice I did that. Just habits, I think. However, I thought I saw the same changes in @vesaakerman's [PR #35](https://github.com/DANS-KNAW/easy-validate-dans-bag/pull/35/files#diff-a22c91cb3d78d31d392dcfa6c05646edR279)